src/backend/src/utils/validation.utils.ts

```diff
@@ -64,23 +64,19 @@ const wbsProposedChangesExists = (validationObject: ValidationChain): Validation
   return validationObject.if((value: any, { req }: any) => req.body.wbsProposedChanges);
 };
 
-export const wbsProposedChangesValidators = [
-  body('wbsProposedChanges').optional(),
-  nonEmptyString(wbsProposedChangesExists(body('wbsProposedChanges.name'))),
-  isStatus(wbsProposedChangesExists(body('wbsProposedChanges.status'))),
-  wbsProposedChangesExists(body('wbsProposedChanges.links')).isArray(),
-  nonEmptyString(body('wbsProposedChanges.links.*.url')),
-  nonEmptyString(body('wbsProposedChanges.links.*.linkTypeName')),
-  intMinZero(body('wbsProposedChanges.projectLeadId').optional()),
-  intMinZero(body('wbsProposedChanges.projectManagerId').optional())
-];
-
 const projectProposedChangesExists = (validationObject: ValidationChain): ValidationChain => {
   return validationObject.if((value: any, { req }: any) => req.body.projectProposedChanges);
 };
 
 export const projectProposedChangesValidators = [
   body('projectProposedChanges').optional(),
+  nonEmptyString(wbsProposedChangesExists(body('projectProposedChangesExists.name'))),
```

Comment:
if you go look at this helper method, it's basically saying, only validate this body item based on these things if this thing is true (in this case, if the body has wbsProposedChanges). Because we're eliminating the wbsProposedChanges field in the route, you should use projectProposedChangesExists in its place for the projectProposedChangesValidators and workPackageProposedChangesExists in its place for the workPackageProposedChangesValidators